recipe/meta.yaml

```diff
@@ -28,35 +29,35 @@ requirements:
     - beautifulsoup4
     - cartopy >=0.17.0
     - cartopy_offlinedata
-    - cdms2 3.1.5
-    - cdutil 8.2.1
-    - dask
-    - esmpy >=8.4.0
-    - genutil 8.2.1
+    - cf-units
+    - dask !=2024.12.0,!=2024.12.1
     - lxml
     - mache >=0.15.0
-    - matplotlib-base
+    - matplotlib-base >=3.8.2
     - netcdf4
     - output_viewer >=1.3.0
     - numpy >=2.0.0,<3.0.0
+    - pywavelets
     - scipy
-    - shapely >=2.0.0,<3.0.0
-    - xarray >=2024.03.0
+    - xarray >=2024.3.0
+    - xcdat >=0.7.3,<1.0.0
+    - xesmf >=0.8.7
+    - xskillscore >=0.0.20
 
 test:
   requires:
     - python {{ python_min }}
     - pip
-    - pytest
-    - pytest-cov
-  source_files:
-    - tests/*
+  #  - pytest
+  #  - pytest-cov
+  # source_files:
+  #  - tests/*
```

Comment:
```suggestion
```

Reply:
The integration test suite is expensive and throttles the LCRC websever. We disabled testing on the `dev` branch for release candidates.

I don't think we need to keep these commented out code around.